client/extensions/woocommerce/app/stats/stats-chart/index.js

```diff
@@ -1,25 +1,96 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
+import classnames from 'classnames';
+import page from 'page';
 
 /**
  * Internal dependencies
  */
+import Card from 'components/card';
 import QuerySiteStats from 'components/data/query-site-stats';
 import { getSiteStatsNormalizedData, isRequestingSiteStatsForQuery } from 'state/stats/lists/selectors';
+import ElementChart from 'components/chart';
+import Legend from 'components/chart/legend';
 
-const StatsChart = ( { siteId, query } ) => {
-	return (
-		<div>
-			{ siteId && <QuerySiteStats
-				siteId={ siteId }
-				statType="statsOrders"
-				query={ query }
-			/> }
-		</div>
-	);
+class StatsChart extends Component {
+	state = {
+		selectedTabIndex: 0
+	}
+
+	barClick = bar => {
+		page.redirect( `${ this.props.path }?startDate=${ bar.data.period }` );
+	}
+
+	buildChartData = ( item, selectedTab ) => {
+		const { selectedDate } = this.props;
+		const className = classnames( item.classNames.join( ' ' ), {
+			'is-selected': item.period === selectedDate,
+		} );
+		let value;
+		switch ( selectedTab.attr ) {
+			case 'total_sales':
+				value = item.total_sales;
+				break;
+			case 'net_sales':
+				value = Number( item.total_sales ) - Number( item.total_tax );
+				break;
+			case 'orders':
+				value = item.orders;
+				break;
+			case 'order_average':
+				value = item.orders === '0' ? 0 : Number( item.products ) / Number( item.orders );
```

Comment:
I think you are calculating 'Items Per Order' here and you should be calculating 'Average Order Value':

```
const avgOrderValue = Math.round( ( Number( item.total_sales) / Number( item.orders ) )  * 100 ) / 100;
```